apps/api/src/app/app.module.ts

```diff
@@ -333,7 +333,6 @@ const environment = getConfig
         NationalRegistryClientConfig,
         SyslumennClientConfig,
         ElectronicRegistrationsClientConfig,
-        FeatureFlagConfig,
```

Comment:
We should just leave the gql api alone. And Gulli is also going to add feature flag to the user profile domain, so if it was not using this config it will be used soon.

Reply:
This is being used already and it was double imported. It cant hurt to fix this :)
![image](https://github.com/island-is/island.is/assets/112904566/135a7c41-c2dc-4aae-bcde-5bfd8947111a)